src/foam/core/PropertyInfo.java

```diff
@@ -43,7 +44,8 @@
   public Parser queryParser();
   public Parser csvParser();
   public void toJSON(foam.lib.json.Outputter outputter, Object value);
-  public void toCSV(foam.lib.csv.Outputter outputter, Object value);
+  public void toCSV(X x, Object obj, foam.lib.csv.CSVOutputter outputter, Object value);
+  public void toCSVLabel(foam.lib.csv.CSVOutputter outputter, Object value);
```

Comment:
I don't think you need the `value` in this method at all do you?

Reply:
No we don't. Thx for the reminder